### ISSUE TITLE
Problem with oa_fetch

### ISSUE THREAD

**Issue by JeffreySmithA** [ropensci/openalexR#156]
Hi (again!),

Similar to the thread below, I'm facing a problem with oa_fetch.

For example:

timothy_jackson_oa <- oa_fetch(
  author.id = "A5015292781",
  entity = "works",
  verbose = TRUE
)

I face an error:

Requesting url: https://api.openalex.org/works?filter=author.id%3AA5015292781
Getting 1 page of results with a total of 74 records...
Error in (function (..., deparse.level = 1, make.row.names = TRUE, stringsAsFactors = FALSE,  : 
  numbers of columns of arguments do not match

Before, this function worked perfectly well but now I'm running into an error.

**Comment by mancebral** [comment]
> Hi (again!),
> 
> Similar to the thread below, I'm facing a problem with oa_fetch.
> 
> For example:
> 
> timothy_jackson_oa <- oa_fetch( author.id = "A5015292781", entity = "works", verbose = TRUE )
> 
> I face an error:
> 
> Requesting url: https://api.openalex.org/works?filter=author.id%3AA5015292781 Getting 1 page of results with a total of 74 records... Error in (function (..., deparse.level = 1, make.row.names = TRUE, stringsAsFactors = FALSE, : numbers of columns of arguments do not match
> 
> Before, this function worked perfectly well but now I'm running into an error.

Yes, same error with oa_fetch; but as long as I remember this function worked a month ago providing "tibble" or "dataframes" not just "lists".

**Comment by JeffreySmithA** [comment]
Hey Manuel,

Sorry I didn't follow your comment. How would I change the function make it work?

Thanks!

**Comment by yjunechoe** [comment]
Thanks for the report! I can reproduce this error.

Looks like the error popped up due to the openalex database adding a "institution lineage" to authors yesterday. But the coverage of this field is spotty, so some authors have "institution lineage" this while others don't. I'll draft a PR to address this new development

**Comment by mancebral** [comment]
> Hey Manuel,
> 
> Sorry I didn't follow your comment. How would I change the function make it work?
> 
> Thanks!

Yes, finally it works intalling the version of the package you left on GitHub !!!

**Comment by trangdata** [comment]
Hi @JeffreySmithA yes this issue is similar to #154 and the fix in #155 has been merged. Could you please try re-installing the package from GitHub and try your `oa_fetch` call again? 🙏🏽 

```r
# install.packages("remotes")
remotes::install_github("ropensci/openalexR")
```

**Comment by JeffreySmithA** [comment]
Yes, this worked - much appreciated!
